docs/node-operators/archive-nodes/index.mdx

```diff
@@ -6,7 +6,7 @@ keywords:
   - archive node
 ---
 
-# Archive Node
+# About Archive Nodes
```

Comment:
oh, I see what you did here! (another nested `archive-nodes` section).... hmmm
docs search works best on only two levels of nesting

Okay, there are opportunities here to remediate the impact

with this change, all links to /node-operators/archive-node
must change to /node-operators/archive-nodes

If you keep the plural `archive-nodes` directory, be sure to put in a redirect so that links published outside of the docs (for example, blog posts and bookmarks) will still work - see https://github.com/o1-labs/docs2/wiki/Redirects-on-the-docs-website 

Suggestion: rename the directory to singular `archive-node` so the many links throughout the docs still work (the magic of index.mdx files!) 

For this reason, the preferred syntax for links uses the shortened path (not the relative path) to simplify maintenance 
see https://github.com/o1-labs/docs2/wiki/Links#links-to-internal-urls 

I can't check out your forked branch... so you'll need to test doc builds and links on your end

Reply:
you've changed it archive-node - if you want that that's ok but I'd ask you to sort it out.

Reply:
@halsaphi  
You'll need to do the directory rename on your end, I can change files, but not directories

Just right-click `archive-nodes` in the left pane and rename to `archive-node`